src/lib.rs

```diff
@@ -197,6 +198,7 @@ pub fn readback_hvm_net(
   let mut diags = Diagnostics::default();
   let net = hvm_to_net(net);
   let mut term = net_to_term(&net, book, labels, linear, &mut diags);
+  #[allow(clippy::mutable_key_type)]
```

Comment:
Did this start to flag after a clippy update?

Reply:
I think so, with rust 1.80